nikola/plugins/template/jinja.py

```diff
@@ -52,7 +52,10 @@ def __init__(self):
         """Initialize Jinja2 environment with extended set of filters."""
         if jinja2 is None:
             return
-        self.lookup = jinja2.Environment()
+        # Not ideal because it's not using our cache structure
```

Comment:
Where is it held then? If this is the case, couldn’t we handle this in `create_lookup` or `set_site`?